resources/compass/charts/gateway/templates/gateway.yaml

```diff
@@ -34,4 +25,25 @@ spec:
         httpsRedirect: true # automatic 301 redirect from http to https
       hosts:
         - "*.{{.Values.global.ingress.domainName}}"
+
+---
+
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
To make things simpler those Gateways should be installed always so we do not need `.Values.gateway.enabled` which is switched depending if we install Compass with or without Core.

Reply:
modified as agreed.